content/en/logs/archives/reloading/s3.md

```diff
@@ -0,0 +1,151 @@
+            "Resource": [
+                "arn:aws:s3:::<MY_BUCKET_NAME_1_/_MY_OPTIONAL_BUCKET_PATH_1>/*",
+                "arn:aws:s3:::<MY_BUCKET_NAME_2_/_MY_OPTIONAL_BUCKET_PATH_2>/*"
+            ]
+        },
+        {
+            "Sid": "DatadogReloadLogArchivesListBucket",
+            "Effect": "Allow",
+            "Action": "s3:ListBucket",
+            "Resource": [
+                "arn:aws:s3:::<MY_BUCKET_NAME_1>",
+                "arn:aws:s3:::<MY_BUCKET_NAME_2>"
+            ]
+        },
+        {
+            "Sid": "DatadogReloadLogArchivesGetObject",
+            "Effect": "Allow",
+            "Action": "s3:GetObject",
+            "Resource": [
+                "arn:aws:s3:::<MY_BUCKET_NAME_1_/_MY_OPTIONAL_BUCKET_PATH_1>/*",
+                "arn:aws:s3:::<MY_BUCKET_NAME_2_/_MY_OPTIONAL_BUCKET_PATH_2>/*"
+            ]
+        }
+    ]
+}
+```
+
+## Updating Legacy S3 Archives for Reloading
+
+Originally, Datadog's AWS S3 archiving feature depended on your S3 bucket policy to grant appropriate write access to your S3 bucket (this is the industry standard for S3 log archival). But for added scurity, Datadog now writes to your AWS S3 log archives using the IAM Role in your AWS account that you configured for [your AWS integration][2]. 
+
+Datadog only supports reloading from archives that have been configured to use role delegation. Once you have modified your Datadog IAM role to include the IAM policy above, simply edit your archive in the Log Configuration page, select which AWS Account + Role combination you wish to write and reload your archives, and then save your archive. 
+
+## Starting an Archive Reload Job
+
+To start reloading log events from your S3 archives, go to the [Historical Views tab][4] of the [Logs Configuration page][5] in your Datadog account. Select [the "Reload from Archives" button][6]. 
+
+{{< img src="logs/archives/log_historical_views.png" alt="Reload from Archives" responsive="true" style="width:75%;">}}
+
+Complete the reload form and select "Reload from Archives" to initiate the reload job. The reloaded results will go to their own special index called a "Historical View" and will have all existing facets applied. If you wish to reload from an archive that no longer exists, [follow these instructions to create that archive again][1].
+
+{{< img src="logs/archives/log_reload_from_archives.png" alt="Reload from Archives" responsive="true" style="width:75%;">}}
+
+Once you have started a reload job, the job will appear as "pending" in [the Historical Views tab][4]. When it completes it will appear as "active" and will link you to the log explorer where you can find the reloaded log events.
+
+How long a reload job takes to completed depends on the volume of data that exists in the archive for the time-range queried on, and on the count of log events that match the reload query. Reloads over large archives or large time periods can take hours to complete.
+
+## Private Beta Limitations
```

Comment:
Not sure we put beta limitations in the documentation. Would have to check with the documentation team.